mod-orgs/examples/organization_put.sample

```diff
@@ -157,6 +157,11 @@
       "notes": ""
     }
   ],
+  "tags" : {
```

Comment:
I think PUT sample file is the same as POST sample, so it's redundant and do not required here at all.

Reply:
This file was added not with the current PR. I don't think that's a reason to hold up the merging.